src/main/resources/org/openstreetmap/atlas/geography/atlas/items/complex/aoi/aoi-tag-filter.json

```diff
@@ -0,0 +1,20 @@
+{
+  "filters": [
+    "aerialway->STATION",
```

Comment:
Here each line usually represents an "AND" if you use `ConfiguredTaggableFilter`. I see that you chose to not use that class and make every line an "OR" (using stream.anyMatch). It would be nicer to use ConfiguredTaggableFilter here, but it would mean adding all those filters on one line, with OR statements (`|`), and the whole thing would be very unreadable. So I guess it makes sense as is :)

Reply:
It is the same reasons why I did not use the ConfiguredTaggableFilter. As there is a good chance we may want to update the filters in the future, maintaining the filters in new line would make it readable and easy to maintain as opposed to adding all the filters in a single line with OR statements.